projects/storefrontstyles/scss/components/product/carousel/_carousel.scss

```diff
@@ -159,4 +159,20 @@
       min-width: 48px;
     }
   }
+
+  @include cx-highContrastTheme {
+    .indicators {
+      button {
+        background-color: var(--cx-color-background);
+      }
+    }
+    .indicators button[disabled] .cx-icon {
+      color: var(--cx-color-dark);
+      background-color: var(--cx-color-light);
+      padding: 4px;
```

Comment:
`query:` is it fine to have a "magic" value here instead of a variable? should it be easily customizable?

Reply:
I think we should create global sizing variables to standardize the entire application, but not in this PR.